src/index.css

```diff
@@ -74,3 +74,137 @@ body {
   max-width: 70%;
   height: auto;
 }
+
+.container {
+  width: 88%;
+  height: auto;
+  margin: 16px;
+
+  /* border: 1px solid #ccc; */
+  box-shadow: rgba(0, 0, 0, 0.56) 0 22px 70px 4px;
+  padding: 16px;
+  margin-left: 6%;
+}
+
+.list {
+  padding: 20px;
+  box-shadow: rgba(0, 0, 0, 0.1) -4px 9px 25px -6px;
+}
+
+.list .book {
+  column-span: all;
+  padding: 12px;
+
+  /* border: 1px solid black; */
+  box-shadow: rgba(0, 0, 0, 0.16) 0 1px 4px;
+  margin-bottom: 20px;
+  text-align: center;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.book .left {
+  float: left;
+}
+
+.book .clr {
+  clear: both;
+}
+
+.title {
+  font-size: 30px;
+  font-weight: bold;
+}
+
+.remove {
+  background: linear-gradient(to bottom right, #ef4765, #ff9a5a);
+  border: 0;
+  border-radius: 12px;
+  color: #fff;
+  cursor: pointer;
+  display: inline-block;
+  font-family: -apple-system, system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Comment:
- [ ] The Font family is not this in the original design. Kindly check the original [design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) and update your design.

Reply:
@ahmetbozaci Done